### PR TITLE
Update Redis Chinese doc

### PR DESCRIPTION
**Pull request by acelyc111**

### CODE REVIEW

**Comment by acelyc111** [comment]
https://github.com/apache/incubator-pegasus/issues/1726

File: _docs/zh/api/redis.md [hunk 1]
```diff
@@ -2,41 +2,47 @@
...
 
-在pegasus上添加了redis proxy后，用户可以通过redis协议直接访问proxy，从而间接访问pegasus服务。整体架构如下：
+在 Pegasus 集群中部署 Redis Proxy 组件后，用户可以通过 Redis 协议直接访问 Redis Proxy，从而间接访问 Pegasus 服务，从而实现使用 Redis 客户端访问 Pegasus 服务的目的。
+
+整体架构如下：
 
 ![redis_proxy_arch.png](/assets/images/redis_proxy_arch.png){:class="img-responsive"}
 
-redis客户端与redis proxy之间使用[redis协议](https://redis.io/topics/protocol)，目前proxy已支持所有redis 数据类型（Simple Strings、Errors、Integers、Bulk Strings、Arrays）。
+Redis 客户端与 Redis Proxy 之间使用[redis协议](https://redis.io/topics/protocol)，目前 proxy 已支持所有的 RESP2 [协议](https://redis.io/docs/reference/protocol-spec/)数据类型（即：Simple Strings、Errors、Integers、Bulk Strings、Arrays）。
```
Comment (empiredan) [inline _docs/zh/api/redis.md:15]:
```suggestion
Redis 客户端与 Redis Proxy 之间使用[redis协议](https://redis.io/topics/protocol)，目前 Proxy 已支持所有的 RESP2 [协议](https://redis.io/docs/reference/protocol-spec/)数据类型（即：Simple Strings、Errors、Integers、Bulk Strings、Arrays）。
```

File: _docs/zh/api/redis.md [hunk 2]
```diff
@@ -2,41 +2,47 @@
...
 
-redis proxy与pegasus集群之间使用pegasus的thrift协议，proxy在这里就类似一个普通的pegasus client，从meta server查询meta信息、与replica server进行用户数据的读写。
+Redis Proxy 与 Pegasus 集群之间使用 Pegasus 的协议，proxy 在这里就类似一个普通的 Pegasus client，从 Meta Server 查询路由信息、与 Replica Server 进行用户数据的读写。
```
Comment (empiredan) [inline _docs/zh/api/redis.md:17]:
```suggestion
Redis Proxy 与 Pegasus 集群之间使用 Pegasus 的协议，Proxy 在这里就类似一个普通的 Pegasus Client，从 Meta Server 查询路由信息、与 Replica Server 进行用户数据的读写。
```

File: _docs/zh/api/redis.md [hunk 3]
```diff
@@ -2,41 +2,47 @@
...
 
 ## 提供服务的形式
 
-跟redis服务一样，以``host:port``形式提供，如果服务压力大，可以提供多个``host:port``来避免单点proxy压力过大造成瓶颈。当提供多个redis proxy地址时，由于后端访问的都是同一个集群的同一张表，数据是完全相同的。用户可以采用round robin, hash等方式进行负载均衡。
+跟 Redis 服务一样，proxy 实例以 `host:port` 形式提供。如果服务压力大，可以提供多个 proxy 实例，通过水平扩展的方式来提升服务吞吐量。
```
Comment (empiredan) [inline _docs/zh/api/redis.md:21]:
```suggestion
跟 Redis 服务一样，Proxy 实例以 `host:port` 形式提供。如果服务压力大，可以提供多个 Proxy 实例，通过水平扩展的方式来提升服务吞吐量。
```

File: _docs/zh/api/redis.md [hunk 4]
```diff
@@ -2,41 +2,47 @@
...
+
+Proxy 是无状态的，多个 proxy 实例共享同一个后端 Pegasus 服务。可以采用round robin, hash等方式进行负载均衡。
```
Comment (empiredan) [inline _docs/zh/api/redis.md:23]:
```suggestion
Proxy 是无状态的，多个 Proxy 实例共享同一个后端 Pegasus 服务。可以采用round robin, hash等方式进行负载均衡。
```

File: _docs/zh/api/redis.md [hunk 5]
```diff
@@ -2,41 +2,47 @@
...
->proxy的可执行文件为``pegasus_rproxy``, 由``./run.sh pack_tools``打包生成。
+> proxy 的可执行文件为 `pegasus_rproxy`, 由 `./run.sh pack_tools` [打包](/docs/build/compile-by-docker/#packaging)生成。
 
 ## 配置
 
-redis proxy的配置文件规则遵循[配置说明](/administration/config)，参考[示例](https://github.com/apache/incubator-pegasus/blob/master/src/geo/bench/config.ini)。
+Redis Proxy 的配置文件规则遵循[配置说明](/administration/config)，参考[示例](https://github.com/apache/incubator-pegasus/blob/master/src/redis_protocol/proxy/config.ini)。
 
-在redis proxy中有几项特有的配置项需要注意：
+在 proxy 中有几项特有的配置项需要注意：
```
Comment (empiredan) [inline _docs/zh/api/redis.md:31]:
```suggestion
在 Proxy 中有几项特有的配置项需要注意：
```

File: _docs/zh/api/redis.md [hunk 6]
```diff
@@ -2,41 +2,47 @@
...
 
 ## 提供服务的形式
 
-跟redis服务一样，以``host:port``形式提供，如果服务压力大，可以提供多个``host:port``来避免单点proxy压力过大造成瓶颈。当提供多个redis proxy地址时，由于后端访问的都是同一个集群的同一张表，数据是完全相同的。用户可以采用round robin, hash等方式进行负载均衡。
+跟 Redis 服务一样，proxy 实例以 `host:port` 形式提供。如果服务压力大，可以提供多个 proxy 实例，通过水平扩展的方式来提升服务吞吐量。
+
+Proxy 是无状态的，多个 proxy 实例共享同一个后端 Pegasus 服务。可以采用round robin, hash等方式进行负载均衡。
 
->proxy的可执行文件为``pegasus_rproxy``, 由``./run.sh pack_tools``打包生成。
+> proxy 的可执行文件为 `pegasus_rproxy`, 由 `./run.sh pack_tools` [打包](/docs/build/compile-by-docker/#packaging)生成。
```
Comment (empiredan) [inline _docs/zh/api/redis.md:25]:
```suggestion
> Proxy 的可执行文件为 `pegasus_rproxy`, 由 `./run.sh pack_tools` [打包](/docs/build/compile-by-docker/#packaging)生成。
```

**Review by empiredan — APPROVED** [review]

**Pull request merged**